### DISCUSSION TITLE
Trying to use HTTP Server with Webview

### DISCUSSION THREAD

**Discussion by wwderw** [denoland/deno#8976] [General]
I've got some programs that need to be used in conjunction with a simple webserver.

With node, I would use express and webview.

With these 2 in node, I would use(just a portion of my entry js code):

`const app = express()

app.get("/", (req, res) => {
res.send("./index.html")
});

app.listen(8000)`

and for the url in the webview, I would have:

'url: "http://localhost:8000/index.html"'

and it would work flawlessly.

Apparently I am not finding the correct app.get() to send the file for webview to pick it up.

Any thoughts on that? Just needing something to bootstrap the html to send to webview to help satisfy CORs for some of my programs.  Or if there is a better way to do it then that, I would appreciate any help in that direction.  The only thing that I would like to stick with is webview as I prefer sending to that then sending to a regular browser.

Thanks

**Comment by help-14** [comment]
@wwderw Hi, how did you get pass this problem?
